### PR TITLE
resolves #37 explain how to integration with Antora

### LINKED ISSUE #37: Document how to use with Antora
This looks like a real gem 💎 !

But how do you use it with Antora? I installed Antora with `npm` and have been happily running `antora antora-playbook.yml` so far. What do I need to change to have something simple as 

```
[plantuml,alice-bob,svg,role=sequence]
....
alice -> bob
....
```

inside an `.adoc` rendered as a diagram?

### PR DESCRIPTION
**Pull request by Mogztter**
resolves #37

//cc @akosma @marcelstoer @danyill review highly appreciated! :smile:

### CODE REVIEW

**Comment by marcelstoer** [comment]
> resolves #10

Shouldn't this resolve #37 instead?

**Comment by Mogztter** [comment]
Yes you're right, nice catch! I don't know where the 10 comes from... I will update the commit 😅

File: README.md [hunk 1]
```diff
@@ -117,6 +117,36 @@ You can also set this attribute using the Javascript API, for instance:
...
+      extensions:
+        - asciidoctor-kroki
+    ```
+
+    https://docs.antora.org/antora/2.2/playbook/configure-asciidoc/#extensions
+
+3. Enjoy!
+
+**💡 TIP**:
+You can use the `kroki-fetch-diagram` option to download the images from Kroki during the build time.
```
Comment (marcelstoer) [inline README.md:141]:
```suggestion
You can use the `kroki-fetch-diagram` option to download the images from Kroki during at build time.
```

**Comment by marcelstoer** [comment]
All good, thanks.

File: README.md [hunk 2]
```diff
@@ -117,6 +117,36 @@ You can also set this attribute using the Javascript API, for instance:
...
+    asciidoc:
+      extensions:
+        - asciidoctor-kroki
+    ```
+
+    https://docs.antora.org/antora/2.2/playbook/configure-asciidoc/#extensions
+
+3. Enjoy!
+
+**💡 TIP**:
```
Comment (marcelstoer) [inline README.md:140]: Once you address #39 you an add a reference to that new chapter (I suppose) here.

Comment (Mogztter) [reply README.md:141]: Yes, I will also add a table with a list of options

**Comment by Mogztter** [comment]
Thanks for your review @marcelstoer 
I've updated the commit and apply your suggestion :+1:

**Pull request merged**